velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -60,6 +60,17 @@ core::SortOrder toSortOrder(const ::substrait::SortField& sortField) {
   }
 }
 
+std::vector<core::TypedExprPtr> NamesToFieldAccessExpressions(
```

Comment:
Is this method used? If it is, please, fix naming.

Reply:
Sure, I will update.